### ISSUE TITLE
UI issue 

### ISSUE THREAD

**Issue by Davinel17** [War3zuk/War3zuk-Alpha-18-Stable-AIO#8]
i got a error with the SMXui  and the war3zuk UI 

i got alot of error like that  2020-06-11T20:38:02 121.010 WRN XML patch for "XUi/xui.xml" from mod "War3zuk Alpha 18 AIO" did not apply: <set
like around 30 to 50 error like this one  

i dont know if is the UI of the mod (yours) the ZMXuiWar3zukCP or the SMXui or mayby i need to combine some  i dont know 

thank you if you helping me .

**Comment by War3zuk** [comment]
If using the smxui you need the patch too... there made by sirillion

**Comment by Davinel17** [comment]
Yeah is that i think too , Unfortunately he dont make any patch for that. the last was for 18.3, anyway

thanks for your anwser and thank you for your great job :)

**Comment by War3zuk** [comment]
https://github.com/Sirillion/ZMXuiWar3zukCP

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
